src/pages/Login.jsx

```diff
@@ -35,6 +36,7 @@ export const Login = () => {
             email: "testing@test.com",
```

Comment:
Suggestion- it's a good practice to use keys/ credentials using .env file.

Reply:
ok, will look into it.